sensei-course-progress.php

```diff
@@ -16,6 +16,36 @@
 
 if ( ! defined( 'ABSPATH' ) ) exit;
 
+/**
+ * Sensei Detection
+ */
+function sensei_course_progress_init(){
```

Comment:
This function needs to have the whitespace fixed up as per the [PHP coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage).